enterprise/views.py

```diff
@@ -1178,6 +1184,15 @@ def get(self, request, enterprise_uuid, program_uuid):
             # The user is already enrolled in the program, so redirect to the program's dashboard.
             return redirect(LMS_PROGRAMS_DASHBOARD_URL.format(uuid=program_uuid))
 
+        # Check to see if access to any of the course runs in the program are restricted for this user.
+        course_run_ids = []
+        for course in program_details['courses']:
+            for course_run in course['course_runs']:
+                course_run_ids.append(course_run['key'])
```

Comment:
Any edX recommended guidance on for loops versus list comprehensions?
ie. I think this something like this would work as well but might be less readable: `course_run_ids = [course_run['key'] for course in program_details['courses'] for course_run in course['course_runs']]`

Reply:
Yeah, I would op for readability in this case.